cmd/zfs/zfs_main.c

```diff
@@ -4261,6 +4261,10 @@ zfs_do_send(int argc, char **argv)
 		case 'P':
 			flags.parsable = B_TRUE;
 			break;
+		case 'V':
```

Comment:
I'd suggest adding a comment here that `-V` will be silently ignored on platforms where it's unsupported.